Project/src/MakeCall/Lobby.js

```diff
@@ -31,34 +41,52 @@ const Lobby = ({ call }) => {
         }
     };
 
-    const admitAllParticipants = async () => {
+    capabilitiesChangedHandler = (capabilitiesChangeInfo) => {
+        console.log('lobby:capabilitiesChanged');
+        for (const [key, value] of Object.entries(capabilitiesChangeInfo.newValue)) {
+            if(key === 'manageLobby' && value.reason != 'FeatureNotSupported') {
+                (value.isPresent) ? this.setState({ canManageLobby: true }) : this.setState({ canManageLobby: false });
+                const admitAllButton = document.getElementById('admitAllButton');
+                if(this.state.canManageLobby === true){
+                    admitAllButton.style.display = '';
+                } else {
+                    admitAllButton.style.display = 'none';
+                }
+                continue;
+            }
+        }
+    };
+
+    async admitAllParticipants() {
         console.log('admitAllParticipants');
         try {
-            await lobby.admitAll();
+            await this.lobby?.admitAll();
         } catch (e) {
             console.error(e);
         }
     }
 
-    return (
-        <div>
-            {
-                (lobbyParticipantsCount > 0) &&
-                <div className="ms-Grid-row">
-                    <div className="ml-2 inline-block">
-                        <div>In-Lobby participants number: {lobbyParticipantsCount}</div>
-                    </div>
-                    <div className="ml-4 inline-block">
-                        <PrimaryButton className="primary-button"
-                                        iconProps={{ iconName: 'Group', style: { verticalAlign: 'middle', fontSize: 'large' } }}
-                                        text="Admit All Participants"
-                                        onClick={admitAllParticipants}>
-                        </PrimaryButton>
+    render() {
+        return (
+            <div>
+                {
+                    (this.state.lobbyParticipantsCount > 0) &&
+                    <div className="ms-Grid-row">
+                        <div className="ml-2 inline-block">
+                            <div>In-Lobby participants number: {this.state.lobbyParticipantsCount}</div>
+                        </div>
+                        <div className="ml-4 inline-block">
+                            <PrimaryButton className="primary-button"
+                                            id="admitAllButton"
+                                            style={{ display: this.state.canManageLobby ? '' : 'none' }}
```

Comment:
with this style will the admitAll button not appear at all? or hidden?

can you share a screen shot of capable vs not capable UI experience

Reply:
The button will be hidden if canManageLobby is false. Will attach the screen shot later

Reply:
When canManageLobby is true: 
<img width="556" alt="image" src="https://github.com/Azure-Samples/communication-services-web-calling-tutorial/assets/102097103/7d7ecb33-2c52-4aab-877a-f78bf61629b7">

Reply:
When canManageLobby is false:
<img width="556" alt="image" src="https://github.com/Azure-Samples/communication-services-web-calling-tutorial/assets/102097103/9fd5392c-ce84-4f50-8b18-dbbf4ef24a0c">